config/config.proto

```diff
@@ -34,6 +34,19 @@ message PushoverConfig {
   optional string user_key = 2;
 }
 
+message HipChatConfig {
```

Comment:
Please add a comment above this: `// Configuration for notification via HipChat.`, as with the other messages.